Week2/homework/map-filter.js

```diff
@@ -2,7 +2,9 @@
 
 function doubleOddNumbers(numbers) {
   // Replace this comment and the next line with your code
-  console.log(numbers);
+  const oddNum = numbers.filter(num => num % 2 === 1);
+  const doubleOddNum = oddNum.map(num => num * 2);
+  return doubleOddNum;
```

Comment:
👍 Interesting that you used `num % 2 === 1` instead of `num % 2 !== 0`. Most of the developers are used to second approach but I think your approach also works just fine. 🙂

Reply:
I think for a beginner, what I have written sounds more logical for me or easier to understand. However, as you mentioned, it think I should also use the most popular approach as a future pro. :)